app/Http/Controllers/Api/V1/ProjectHasUserCustodianApprovalController.php

```diff
@@ -0,0 +1,237 @@
+<?php
+
+namespace App\Http\Controllers\Api\V1;
+
+use Exception;
+use App\Http\Controllers\Controller;
+use App\Models\ProjectHasUserCustodianApproval;
+use Illuminate\Http\Request;
+use App\Http\Traits\Responses;
+use App\Models\Custodian;
+use Illuminate\Support\Facades\Gate;
+
+class ProjectHasUserCustodianApprovalController extends Controller
+{
+    use Responses;
+
+
+    /**
+     * @OA\Get(
+     *      path="/api/v1/custodians/{custodianId}/projecUsers/{projectUserId}",
```

Comment:
Spelling mistake "projecUsers", and urls generally need to follow kebab-case or snake_case (although less preferred these days) as there is no casing in urls.